front/app/containers/App/constants.ts

```diff
@@ -114,6 +117,7 @@ export const appGraphqlLocalePairs = {
   nlBe: 'nl-BE',
   nlNl: 'nl-NL',
   deDe: 'de-DE',
+  itIT: 'it-IT',
```

Comment:
Final T shouldn't be capital for the graphql constants

Reply:
Thanks!